### DISCUSSION TITLE
False Notification of New Firmware Version - No Updates Available

### DISCUSSION THREAD

**Discussion by robbieffm** [hoylabs/OpenDTU-OnBattery#1190] [General]
Hello everyone,

I'm regularly receiving a notification that a new software version is available (displayed as "Neue Version verfügbar! Zeige Änderungen!"). However, when I click the link and check on GitHub, there is no released new version available for download.

Here are some system details that might be relevant:

Hostname: OpenDTU-C95788
SDK Version: v4.4.7-dirty
Configuration Version: 0.1.28
Firmware Version / git Hash: 2024.08.18
Firmware Branch: master
PIO Environment: generic_esp32s3

I would like to understand why this notification appears even though there is no new version available on GitHub. Is this a known issue, or could it be something related to my configuration?

Thanks in advance for your help!

![Screenshot 2024-08-23 153431](https://github.com/user-attachments/assets/75c0e9a2-00ae-41c6-9781-4042e5c08995)

**Comment by schlimmchen** [comment]
Fixed in b115f946.

**Reply by robbieffm** [reply]
Thanks :)
